backend/app/api/v1/commons/utils.py

```diff
@@ -133,3 +134,25 @@ def buildReleaseStreamFilter(input_array):
         )
         mapped_array.append(match)
     return list(set(mapped_array))
+
+
+def get_dict_from_qs(query_string):
+    query_dict = parse_qs(query_string)
+    cleaned_dict = {
+        key: [v.strip("'") for v in values] for key, values in query_dict.items()
+    }
+
+    return cleaned_dict
+
+
+def construct_query(filter_dict):
+    query_parts = []
+    if isinstance(filter_dict, dict):
+        for key, values in filter_dict.items():
+            k = constants.FIELDS_FILTER_DICT[key]
+            if len(values) > 1:
+                or_clause = " or ".join([f'{k}="{value}"' for value in values])
```

Comment:
Elsewhere you capitalized the Splunk keyword `OR`. It'd be a bit easier to read if you're consistent about such conventions. The all-caps *does* make it stand out, and is probably worthwhile.

Reply:
Gotcha!